src/components/templates/Jazzbar/JazzTab/JazzTab.scss

```diff
@@ -11,35 +28,46 @@
 
   .music-bar-content {
     display: grid;
-    grid-template-columns: repeat(4, 1fr);
-    padding: 20px 50px;
-    grid-auto-flow: row dense;
+
+    padding: 20px 70px;
     justify-content: center;
     grid-gap: 0.5rem;
 
-    .music-bar-top-left-grid-cell {
-      height: 90px;
-      grid-column: 1;
-      grid-row: 1;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-areas:
+      ".  video video ."
+      ".  video video  .";
+
+    .jazzbar-table-component-container {
```

Comment:
These styles appear to be used in `src/components/templates/Jazzbar/components/JazzBarTableComponent/JazzBarTableComponent.tsx`, do they need to be in this file?

If they're custom overrides I wonder if we can have them somewhere clearer feelings/not necessarily as strictly nested as they are here. Would they still work if they were in the 'root' of the block, rather than nested under `music-bar-content`?

Reply:
Moved them